packages/ui-kit/src/components/Counter/Counter.test.tsx

```diff
@@ -31,6 +31,26 @@ const handlers = [
       )
     }
 
+    if (metricName === 'string-value') {
+      return res(
+        ctx.data({
+          counter: {
+            value: 'My string value'
+          }
+        })
+      )
+    }
+
+    if (metricName === 'boolean-value') {
+      return res(
+        ctx.data({
+          counter: {
+            value: true
```

Comment:
The server will always return strings; however, I suppose in "static" mode customer could provide a boolean like this.